uswds/uswds-site
# USWDS-Site: Replace outdated copySetup references with copyTheme

# Summary
Replaced `copySetup` references with `copyTheme`.

## Related issue

Closes #2070

## Preview link

Preview link: [Phase 2 - Compile page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-fix-copySetup/documentation/getting-started/developers/phase-two-compile/#step-6-export-compile-functions)

## Problem statement

The [Phase 2 Compile guide ](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/#step-6-export-compile-functions) lists the `copySetup` gulp task, but it is now called `copyTheme` in `uswds-compile`. 

Link to `copyTheme` in uswds-compile's `gulpfile.js`: https://github.com/uswds/uswds-compile/blob/develop/gulpfile.js#L261

## Solution

Replaced all `copySetup` references with `copyTheme`

## Testing and review

1. Confirm that `copyTheme` is the correct name of the gulp task
1. Confirm that all `copySetup` references have been updated
